site/guide/basics/strings-and-numbers.md

```diff
@@ -1,32 +1,57 @@
 # Strings and Numbers
 
-The basic building blocks of the language are numbers, strings and expressions
+The basic building blocks of the relevance language are numbers, strings and the expressions
 that combine them.
 
+The "Q:" is the relevance query that is being run and the, "A:" is the answer to our query. We will discover that queries can be complex and the answers to a query can be of any type, but more on that soon!
+
+## Strings and Integers
+
+Strings are a set of characters (a-z,0-9,!@#$) that are surrounded by quotes. To start, we will write a relevance string that outputs, "hello world"
 {{#example}}
 Q: "hello world"
 A: hello world
 {{/example}}
 
-This example outputs a string of characters.
-
 Strings aren’t the only primitives:
 
+Integers are numbers (0-9) without a decimal place. Integers are not wrapped in quotes. We can write a relevance query whose answer is a number
 {{#example}}
 Q: 6000
 A: 6000
 {{/example}}
 
-This above example demonstrates an integer. You can also do math:
+## Simple Operators
+The normal operators you'd expect to be able to use with numbers work in the Relevance Language. Plus and Times are valid operations for integars so this works.
```

Comment:
"Plus and Times" should be "Addition and multiplication". "integars" should be "integers".